.github/workflows/bootstrap.yaml

```diff
@@ -64,12 +64,6 @@ jobs:
           - TEST: docs
           - TEST: s3
 
-    # services:
-    #   postgres:
-    #     image: postgres:9.6
-    #   redis:
-    #     image: redis
```

Comment:
Can this be removed?

Reply:
Yes, and it is removed... Did you tag the right line?

Reply:
I meant can you remove the commented out code here?

Reply:
It is doing exactly that. Removing the comment together with the "real instances" of these services in the workflow templates.

Reply:
I guess I need a vacation. Sorry.